index.html

```diff
@@ -112,7 +112,8 @@ <h2>The calculator</h2>
 <form id="annual-tax" onsubmit="return false;">
 <h3>1. Select tax year</h3>
     <select id="year" class="pretty_button pretty_button_regular">
-  <option value="2022" selected>2022</option>
+  <option value="2023" selected>2023</option>
```

Comment:
Please remove the options for 2016 and 2017, since these can no longer be claimed. If you can be bothered, please remove them also from the dictionaries in index.js for tidiness, but this is not essential.